cmake/Modules/FindComputeCpp.cmake

```diff
@@ -358,31 +454,40 @@ endfunction()
 #  Adds a SYCL compilation custom command associated with an existing
 #  target and sets a dependancy on that new command.
 #
-#  targetName : Name of the target to add a SYCL to.
-#  binaryDir : Intermediate directory to output the integration header.
-#  sourceFiles : Source files to be compiled for SYCL.
+#  TARGET : Name of the target to add SYCL to.
+#  BINARY_DIR : Intermediate directory to output the integration header.
```

Comment:
Is this argument necessary? Can't we just use `CMAKE_CURRENT_BINARY_DIR` instead?

Reply:
Probably, assuming that most callers of "add_sycl_to_target" simply put CMAKE_CURRENT_BINARY_DIR in there anyway. I'll do this.